app/user/models.py

```diff
@@ -29,13 +29,27 @@ def __init__(self, username=None, dn=None, firstName=None, surname=None, mail=No
     @staticmethod
     def create(username, givenName, surname, password, mail = None):
         """
-        Create a user and save it.
+        Create an user and save it.
```

Comment:
Should stay "a user".
From http://dictionary.cambridge.org/grammar/british-grammar/determiners/a-an-and-the:
>  Some words that begin with a vowel letter in writing have a consonant sound:
> 
> /ə ju:ˈnaɪtɪd …/ /ə ju:niˈvɜ:sɪti/ /ə wʌn …/
>
> a united group a university a one-year-old child